README.md

```diff
@@ -10,7 +10,8 @@ Requirements:
 1. Run `npm install` from `app/static`
 1. Run `./scripts/run-dependencies.sh` to start a model API
 1. Run `npm run build` from `app/static` to compile front-end dependencies.
-1. Run app from your IDE or by `cd app && ./gradlew :bootRun` to serve the app on port 8080
+
+1. Run app from your IDE or by `cd app && ./gradlew :app:bootRun` to serve the app on port 8080
```

Comment:
I guess should now be `./src/gradlew  -p src :app:bootRun` or `cd src && ./gradlew :app:bootRun`?